### ISSUE TITLE
model used for detection

### ISSUE THREAD

**Issue by xjtuwh** [iVMCL/AOGNets#3]
Hello! I'm interested in this work. I use AOGNets to faster R-CNN as your paper intorduced. I use the code from https://github.com/jwyang/faster-rcnn.pytorch/tree/pytorch-1.0. And I make a aog.py in lib/model/faster_rcnn  you do. The issue is that when I run train_val to train the model end to end, The loss is so high normally. I want to know where is the problem. Is it related to FP-optimizer? If yes, what should I do for this issues?
I'm looking forward to your reply!
Best wishes!
![图片](https://user-images.githubusercontent.com/22558591/63741397-86600e80-c8c7-11e9-9dc3-f762343b06f2.png)
![图片](https://user-images.githubusercontent.com/22558591/63741625-6c72fb80-c8c8-11e9-8c2d-24970331c107.png)

I use the pretrained model in your google driver.

**Comment by tfwu** [comment]
Please check our https://github.com/iVMCL/AttentiveNorm_Detection
